### DISCUSSION TITLE
geforce experience merge audio tracks?

### DISCUSSION THREAD

**Discussion by nifty360s** [mifi/lossless-cut#826] [Q&A] [answered]
Hello, first off I must say I love loss-cut so much. It makes trimming my gaming highlights and uploading them to wherever so so simple. I do have a question though. I use nvidia geforce to record my clips. They have an option to record desktop (gaming) and microphone separately. Whenever I use that option it, of course, creates a separate audio track for my microphone. Is there a way to combine both desktop and microphone audio into the same track upon save/export using lossless-cut?

**Comment by mifi** [comment, marked as answer]
Hi! You can combine the two tracks into the same file (as separate tracks in one file) with losslesscut. You can however not mix two tracks together into one track. Then you would need to use an external tool to do that first. See #607

**Reply by nifty360s** [reply]
This is what I needed to know, thanks so much!
